BetaCameras/ifm/O3D3xx.cs

```diff
@@ -757,44 +679,150 @@ private string RequestMACAddress(string ip)
             return macAddress;
         }
 
-        private void UpdateWorker_RunWorkerCompleted(object sender, RunWorkerCompletedEventArgs e)
+        private string GetValue(Mode mode)
         {
-            clientSocket.Shutdown(SocketShutdown.Both);
-            clientSocket.Disconnect(true);
-            SetConfigurationMode(true);
-            edit.DeleteApplication(applicationId);
-            device.Save();
-            SetConfigurationMode(false);
+            return ((int)mode).ToString();
```

Comment:
Seems like this could just be done inline. And `GetValue()` is a rather meaningless method name so probably best just to do it inline.

Reply:
I had it inline at first. But didn't like the nesting of all the curly braces.
I wanted to do it as a method of the enum, but in c# that seems only possible as extension method with a lot of boilerplate for just one line.
Guess it is back to inline then.